packages/stimulus/src/index.ts

```diff
@@ -3,14 +3,14 @@ import type { JSClient, JSSpan } from "@appsignal/types"
 export function installErrorHandler(appsignal: JSClient, application: any) {
   const prevHandler = application.handleError
 
-  application.handleError = function (
+  application.handleError = function(
     error: Error,
     message: string,
-    detail: { identifier: string }
+    detail: { identifier?: string }
   ) {
     const span = appsignal.createSpan((span: JSSpan) =>
       span
-        .setAction(`${detail.identifier}-controller`)
+        .setAction(detail?.identifier || "[unknown Stimulus controller]")
```

Comment:
This default string is patterned off of the [vue integration](https://github.com/appsignal/appsignal-javascript/blob/ef1265730fe1eef2dbd52bc71027d3ae90a744b6/packages/vue/src/index.ts#L14).